packages/backend/src/config/config.staging.ts

```diff
@@ -16,12 +16,17 @@ export function getStagingConfig(): Config {
       ...(productionConfig.databaseConnection as Knex.PgConnectionConfig),
       application_name: name,
     },
+    tvlReportSync: false,
```

Comment:
```suggestion
    tvlReportSync: true,
```

Is there a reason why we should not sync tvl on staging?

Reply:
Nvm, staging now === activity app.